src/views/auth.py

```diff
@@ -40,7 +40,10 @@ def register():
         user = User.get(form.email.data.strip())  # strip to remove any excess spaces
         if not user:
             hashed_password = generate_password_hash(form.password.data, method='sha256')
-            new_user = User(email=form.email.data, password=hashed_password)
+            if not mongo.db.user.find_one():
+                new_user = User(email=form.email.data, password=hashed_password, active=True, role="ROLE_ADMIN")
+            else:
+                new_user = User(email=form.email.data, password=hashed_password, active=False, role="ROLE_USER")
```

Comment:
The parameter `role` seems to be hardcoded here. (Or do I misinterpret this?)
I am wondering how this would affect different roles a user might have, either now or in the future.
Are there any scenarios you can think of that requires a different user role? 
Is there an alternative way to retrieve the role somehow?

Reply:
This is hardcoded, yes. This is a functionality to add an admin user to the MongoDB, which you cannot do normally, because you need to be an admin (loop). I tested this code, and it makes sure to make an admin, only if the database (mongDB) is empty of users.

Otherwise, make an account, but have a default role. Then people need to e-mail in order for me to activate their account and giving them the proper role. This enables users to create their own account, but cannot access the data without permission from the admin.

Reply:
Ok. In that case it is important to keep `ROLE_USER` exclusive for new registrations and limit permission/access to only the home/login page or something.. If that is indeed the case and different roles exist to give user permission to something, I agree with the implementation.